.github/workflows/test.yml

```diff
@@ -21,62 +21,61 @@ jobs:
 
     name: CKAN ${{ matrix.ckan-version }}
     runs-on: ubuntu-latest
-    container:
-      image: openknowledge/ckan-dev:${{ matrix.ckan-version }}
     services:
-      solr:
-        image: ckan/ckan-solr:${{ matrix.ckan-version }}
       postgres:
         image: ckan/ckan-postgres-dev:${{ matrix.ckan-version }}
         env:
           POSTGRES_USER: postgres
           POSTGRES_PASSWORD: postgres
           POSTGRES_DB: postgres
+        ports:
+          - 5432:5432
         options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
       redis:
-          image: redis:3
+        image: redis:3
     env:
       CKAN_SQLALCHEMY_URL: postgresql://ckan_default:pass@postgres/ckan_test
       CKAN_DATASTORE_WRITE_URL: postgresql://datastore_write:pass@postgres/datastore_test
       CKAN_DATASTORE_READ_URL: postgresql://datastore_read:pass@postgres/datastore_test
       CKAN_SOLR_URL: http://solr:8983/solr/ckan
       CKAN_REDIS_URL: redis://redis:6379/1
       CKAN_SITE_URL: http://test.ckan.net
+      WORKDIR: /__w/ckanext-switzerland-ng/ckanext-switzerland-ng
```

Comment:
that's the workdir that github uses on the docker container for running the tests on. normally it just creates that for us when we tell it to run the tests on a container ([here](https://github.com/opendata-swiss/ckanext-switzerland-ng/pull/315/files/d056470735279f703546b90abd6aed548848d0b1#diff-faff1af3d8ff408964a57b2e475f69a6b7c7b71c9978cccc8f471798caac2c88L24)). since we are creating our own solr container (to use our custom solr schema), we can't use the github container, we have to create it ourselves. i just copied the commands that github uses to create the container and ran them in the test step, including the WORKDIR, and i put it in a constant here so it was a bit clearer.

Reply:
ok, I see, thank you :)